workflow/python/barebones/v1/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM python:3.8.13
+FROM python:3.10.4-slim-bullseye
 
-RUN apt-get update
+RUN apt-get update && \
+  apt install curl dnsutils unzip git jq libpq-dev python-dev build-essential -y 
```

Comment:
Hey @saurabh0402 - I believe there is a reason we’ve added these to installable. We’re thes missed earlier ?

Reply:
We have changed the base image to the `slim` version which does not have these installed. So, we need to install these.